courses/tests/test_dredd.py

```diff
@@ -0,0 +1,36 @@
+"""
+Test driver for Dredd
+"""
+import os
+import subprocess
+import shlex
+
+from django.conf import settings
+from django.test import LiveServerTestCase
+
+
+class DreddTestCase(LiveServerTestCase):
+    "Tests Dredd related tests."
+    def test_dredd(self):
+        """
+        Run test via Dredd.
+
+        We need to muck with the database url, among other things, to setup the
+        environment for testing properly, since we need to inform the dredd
+        script about our db config.
+        """
+        args = shlex.split('dredd {}/apiary.apib {} -f apiary_hooks.py --language python'.format(
```

Comment:
Would it be possible to treat these arguments as a list all along instead of splitting them here? This may fail in the case where `settings.BASE_DIR` or `live_server_url` contains a space